app/models/webauthn_configuration.rb

```diff
@@ -4,21 +4,37 @@ class WebauthnConfiguration < ApplicationRecord
   validates :credential_id, presence: true
   validates :credential_public_key, presence: true
 
+  def self.roaming_authenticators
+    self.where(platform_authenticator: [nil, false])
+  end
+
+  def self.platform_authenticators
+    self.where(platform_authenticator: true)
+  end
```

Comment:
- I would define the roaming one a `!= true`
- I think the `self.where` is redundant
- I think we can define these as scopes too

```suggestion
  scope :roaming_authenticators, -> { where.not(platform_authenticator: true) }

  scope : platform_authenticators, -> { where(platform_authenticator: true) }
```

Reply:
I think the `where` is redundant, but `!= true` did not work the way I expected. Apparently, `where.not(platform_authenticator: true)` only returns [results that are false and ignores the ones that are null](https://thoughtbot.com/blog/activerecord-s-where-not-and-nil).

Reply:
oh wow `where.not` ... what a mess lol, but the other two parts I think we could do:

```suggestion
  scope :roaming_authenticators, -> { where(platform_authenticator: [nil, false]) }
  scope :platform_authenticators, -> { where(platform_authenticator: true) }
```